source/_integrations/knx.markdown

```diff
@@ -355,7 +355,7 @@ Binary sensors are read-only. To write to the KNX bus configure an exposure [KNX
 ```yaml
 knx:
   binary_sensor:
-    - name: sensor1
+    - name: Sensor 1
```

Comment:
This change is not related to this PR and feature (and a lot of other changes as well). Please do those in a separate PR against the `current` branch. 

Please note; that we have a YAML style guide now, which this change doesn't adhere to either:
<https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

Reply:
regarding the style guide:
> Values that accept a limited set of possible, hardcoded values. For example, mode in automations.

```yaml
sync_state: every 60
```
can be `every | expire` alone or followed by a number (minutes) or `init | true | false`. Or just an integer.
Quoted or not?